src/libraries/System.Private.Xml/src/System/Xml/Xsl/XsltOld/RecordBuilder.cs

```diff
@@ -655,7 +655,7 @@ private void AnalyzeComment()
             }
             else if (minus)
             {
-                _mainNode.ValueAppend(s_Space, false);
+                _mainNode.ValueAppend(s_Space.ToString(), false);
```

Comment:
Just to confirm -- do you think this trade-off is worth it where we had the single-character string getting used both where we could optimize with a `char` and where we need a `string`, and we've not flipped which one we optimize for? Should we consider having `char s_SpaceChar` and `string s_Space`?

Reply:
> do you think this trade-off is worth it

No, if we're adding ToString() calls as a result of this, we're doing something wrong ;-)